<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-TADEQRB" failures="0" tests="28" name="Group_Of_TestCases.Priority" time="11.648" errors="3" timestamp="2024-07-09T16:47:55 IST" skipped="25">
  <testcase classname="Group_Of_TestCases.Priority" name="Browser_Setup" time="11.646">
    <error message="no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=126.0.6478.127)
Build info: version: &#039;4.22.0&#039;, revision: &#039;c5f3146703*&#039;
System info: os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;17.0.9&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [dadfc91568c161f383dda2d3f380196b, maximizeCurrentWindow {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 126.0.6478.127, chrome: {chromedriverVersion: 126.0.6478.126 (d36ace6122e..., userDataDir: C:\Users\hi\AppData\Local\T...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:49904}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:49904/devtoo..., se:cdpVersion: 126.0.6478.127, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: dadfc91568c161f383dda2d3f380196b" type="org.openqa.selenium.NoSuchWindowException">
      <![CDATA[org.openqa.selenium.NoSuchWindowException: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=126.0.6478.127)
Build info: version: '4.22.0', revision: 'c5f3146703*'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.9'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [dadfc91568c161f383dda2d3f380196b, maximizeCurrentWindow {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 126.0.6478.127, chrome: {chromedriverVersion: 126.0.6478.126 (d36ace6122e..., userDataDir: C:\Users\hi\AppData\Local\T...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:49904}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:49904/devtoo..., se:cdpVersion: 126.0.6478.127, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: dadfc91568c161f383dda2d3f380196b
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:518)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:591)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:595)
at org.openqa.selenium.remote.RemoteWebDriver$RemoteWebDriverOptions$RemoteWindow.maximize(RemoteWebDriver.java:1034)
at common_Objects.Objects.url(Objects.java:21)
at Group_Of_TestCases.Priority.Browser_Setup(Priority.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:358)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- Browser_Setup -->
  <testcase classname="Group_Of_TestCases.Priority" name="VerifyHomePageLoads" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyHomePageLoads -->
  <testcase classname="Group_Of_TestCases.Priority" name="SearchBox_Functionality" time="0.000">
    <skipped/>
  </testcase> <!-- SearchBox_Functionality -->
  <testcase classname="Group_Of_TestCases.Priority" name="FilterByCategory" time="0.000">
    <skipped/>
  </testcase> <!-- FilterByCategory -->
  <testcase classname="Group_Of_TestCases.Priority" name="VerifyFilterBySize" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyFilterBySize -->
  <testcase classname="Group_Of_TestCases.Priority" name="VerifyFilterByColor" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyFilterByColor -->
  <testcase classname="Group_Of_TestCases.Priority" name="VerifySortingFunctionalityByPrice" time="0.000">
    <skipped/>
  </testcase> <!-- VerifySortingFunctionalityByPrice -->
  <testcase classname="Group_Of_TestCases.Priority" name="VerifyProductDetailsPage" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyProductDetailsPage -->
  <testcase classname="Group_Of_TestCases.Priority" name="AddToCart" time="0.000">
    <skipped/>
  </testcase> <!-- AddToCart -->
  <testcase classname="Group_Of_TestCases.Priority" name="ViewCartTest" time="0.000">
    <skipped/>
  </testcase> <!-- ViewCartTest -->
  <testcase classname="Group_Of_TestCases.Priority" name="UpdateCartTest" time="0.000">
    <skipped/>
  </testcase> <!-- UpdateCartTest -->
  <testcase classname="Group_Of_TestCases.Priority" name="RemoveProductFromCart" time="0.000">
    <skipped/>
  </testcase> <!-- RemoveProductFromCart -->
  <testcase classname="Group_Of_TestCases.Priority" name="verifyProceedToCheckout" time="0.000">
    <skipped/>
  </testcase> <!-- verifyProceedToCheckout -->
  <testcase classname="Group_Of_TestCases.Priority" name="verifyGuestCheckout" time="0.000">
    <skipped/>
  </testcase> <!-- verifyGuestCheckout -->
  <testcase classname="Group_Of_TestCases.Priority" name="VerifyRegisteredUserCheckout" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyRegisteredUserCheckout -->
  <testcase classname="Group_Of_TestCases.Priority" name="verifyPaymentGatewayIntegration" time="0.000">
    <skipped/>
  </testcase> <!-- verifyPaymentGatewayIntegration -->
  <testcase classname="Group_Of_TestCases.Priority" name="VerifyOrderConfirmationPage" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyOrderConfirmationPage -->
  <testcase classname="Group_Of_TestCases.Priority" name="VerifyOrderHistory" time="0.001">
    <skipped/>
  </testcase> <!-- VerifyOrderHistory -->
  <testcase classname="Group_Of_TestCases.Priority" name="verifyNoResultsForInvalidQueries" time="0.000">
    <skipped/>
  </testcase> <!-- verifyNoResultsForInvalidQueries -->
  <testcase classname="Group_Of_TestCases.Priority" name="verifyNoResultsForInvalidQueries" time="0.000">
    <skipped/>
  </testcase> <!-- verifyNoResultsForInvalidQueries -->
  <testcase classname="Group_Of_TestCases.Priority" name="verifyNoResultsForInvalidQueries" time="0.000">
    <skipped/>
  </testcase> <!-- verifyNoResultsForInvalidQueries -->
  <testcase classname="Group_Of_TestCases.Priority" name="verifyLoginWithValidCredentials" time="0.000">
    <error message="
Found two providers called &#039;credentials&#039; on class Group_Of_TestCases.Priority" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Found two providers called 'credentials' on class Group_Of_TestCases.Priority
at org.testng.internal.Parameters.findDataProvider(Parameters.java:647)
at org.testng.internal.Parameters.findDataProvider(Parameters.java:516)
at org.testng.internal.Parameters.handleParameters(Parameters.java:769)
at org.testng.internal.Parameters.handleParameters(Parameters.java:734)
at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)
at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:946)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- verifyLoginWithValidCredentials -->
  <testcase classname="Group_Of_TestCases.Priority" name="verifyLoginWithInValidCredentials" time="0.000">
    <error message="
Found two providers called &#039;credentials&#039; on class Group_Of_TestCases.Priority" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Found two providers called 'credentials' on class Group_Of_TestCases.Priority
at org.testng.internal.Parameters.findDataProvider(Parameters.java:647)
at org.testng.internal.Parameters.findDataProvider(Parameters.java:516)
at org.testng.internal.Parameters.handleParameters(Parameters.java:769)
at org.testng.internal.Parameters.handleParameters(Parameters.java:734)
at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)
at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:946)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- verifyLoginWithInValidCredentials -->
  <testcase classname="Group_Of_TestCases.Priority" name="Create_Account" time="0.000">
    <skipped/>
  </testcase> <!-- Create_Account -->
  <testcase classname="Group_Of_TestCases.Priority" name="VerifyPasswordResetFunctionality" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyPasswordResetFunctionality -->
  <testcase classname="Group_Of_TestCases.Priority" name="TestWishlistFunctionality" time="0.000">
    <skipped/>
  </testcase> <!-- TestWishlistFunctionality -->
  <testcase classname="Group_Of_TestCases.Priority" name="VerifyProductComparison" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyProductComparison -->
  <testcase classname="Group_Of_TestCases.Priority" name="ProductReviewTest" time="0.001">
    <skipped/>
  </testcase> <!-- ProductReviewTest -->
</testsuite> <!-- Group_Of_TestCases.Priority -->
